changelogs/fragments/271-new-module-iptables_state.yml

```diff
@@ -0,0 +1,5 @@
+---
+minor_changes:
+  - iptables_state - add new module and its action plugin to save iptables
```

Comment:
Please remove this. New modules/action plugins automatically get a changelog entry.